domain.yml

```diff
@@ -15,63 +37,95 @@ entities:
 
 responses:
   utter_greet:
-  - text: "Hi! It's time for your daily wellness check. Tracking healthy habits is a great way to measure your progress over time. Would you like to answer a few questions about your health?"
+  - text: Hi! It's time for your daily wellness check. Tracking healthy habits is a great way to measure your progress over time. Would you like to answer few questions about your health?
+
+  utter_cheer_up:
+  - text: 'Here is something to cheer you up:'
+    image: https://i.imgur.com/nGF1K8f.jpg
+
+  utter_did_that_help:
+  - text: Did that help you?
+
+  utter_happy:
+  - text: Great, carry on!
 
   utter_goodbye:
-  - text: "See you tomorrow. Bye!"
+  - text: Bye
 
   utter_iamabot:
-  - text: "I am a bot, powered by Rasa."
-  
-  utter_ask_confirm_exercise:
+  - text: I am just another bot trying to help you.
```

Comment:
I see the intent and response that's been added for bot_challenge/utter_iamabot, but no rule to create the dialogue?

Reply:
my bad. I am fixing it with below things
**domain.yml**
```yml
  utter_iamabot:
  - text: I am a bot, powered by Rasa.
```

**data/rules.yml**
```yml
- rule: Challenging Rasa Bot
  steps:
  - intent: bot_challenge
  - action: utter_iamabot
```